### PR TITLE
Adjust Project Controller for redux

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by levity** [comment]
is this necessary? see for example how i fetched the posts separately in ProjectPosts

**Comment by robbiecarlton** [comment]
Yeah, this is all getting updated

**Comment by robbiecarlton** [comment]
@levity good to merge?

File: api/controllers/ProjectController.js [hunk 1]
```diff
@@ -16,7 +16,6 @@ var mediaAttributeNames = [
 var projectRelations = [
   {user: qb => qb.column('id', 'name', 'avatar_url')},
   {community: qb => qb.column('id', 'name', 'avatar_url')},
-  {contributors: qb => qb.column('users.id')},
```
Comment (levity) [inline api/controllers/ProjectController.js]: removing this breaks projectAttributes

**Comment by levity** [comment]
:+1:

**Pull request merged**